examples/crossplatform.pp

```diff
@@ -0,0 +1,89 @@
+case $facts['os']['family'] {
```

Comment:
I don't have a puppet dev environment setup right to test, but I believe this should be `facts[:osfamily]` For example

```
describe 'foo' do
  on_supported_os.each do |os, facts|
    context "on #{os}" do
      let(:facts) do
        facts
      end
      if facts[:osfamily] == 'Debian' then
        it { is_expected.to compile }
      end
    end
  end
end
```